utilities/blob_db/blob_db_impl.cc

```diff
@@ -1813,16 +1797,7 @@ Status BlobDBImpl::GCFileAndUpdateLSM(const std::shared_ptr<BlobFile>& bfptr,
       // because
       // a new version of the key came in at this time, which will override
       // the current version being iterated on.
-      if (s1.IsBusy()) {
-        ROCKS_LOG_INFO(db_options_.info_log,
-                       "Optimistic transaction failed delete: %s bn: %" PRIu32,
-                       bfptr->PathName().c_str(), gcstats->blob_count);
```

Comment:
"transaction failed delete" ... this seems like an important log message, isn't it?
Since you have the best knowledge of the code, i'll leave the final decision to you. 

All the messages can be removed in successful cases, but its the non-success messages which are always important to be logged, in my view.